saw-remote-api/src/SAWServer.hs

```diff
@@ -205,6 +208,12 @@ initialState readFileFn =
      halloc <- Crucible.newHandleAllocator
      jvmTrans <- CJ.mkInitialJVMContext halloc
      cwd <- getCurrentDirectory
+     mb_cache <- lookupEnv "SAW_SOLVER_CACHE_PATH" >>= \case
+      Just p | length p > 0 -> do
```

Comment:
I'd phrase this code this way:

```suggestion
      Just p | not (null p) -> do
```

So that this will short-circuit if `p` is an empty string.